client/src/connection/itc/script.ts

```diff
@@ -13,17 +13,23 @@ class SASRunner{
   [System.__ComObject] $objSAS
 
   [void]ResolveSystemVars(){
-  $code =
-@'
-   %let workDir = %sysfunc(pathname(work));
-   %put ${WORK_DIR_START_TAG};
-   %put &workDir;
-   %put ${WORK_DIR_END_TAG};
-   %let rc = %sysfunc(dlgcdir("&workDir"));
-   run;
-'@
-    $this.Run($code)
-    $this.FlushLogLines(4096, $false)
+    try {  
```

Comment:
Looks good. Does it make sense to extract this block into its own PS function in the runner, and just call the function here?

Reply:
I pushed a commit. Does it match what you have in mind?

Reply:
Looks good. Thanks for doing that!